nx-workspace/apps/web/src/components/atoms/core/Snackbar.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import {
+  Snackbar as MSnackbar,
+  SnackbarProps as MSnackbarProps,
+} from "@material-ui/core";
+
+export interface SnackbarProps extends MSnackbarProps {}
+
+export type Snackbar = React.FunctionComponent<SnackbarProps>;
+
+export const Snackbar = (props: SnackbarProps) => <MSnackbar {...props} />;
```

Comment:
Sorry, bad precedent set by my old atoms!

Can replace this with:

```
export { Snackbar, SnackbarProps } from "@material-ui/core";
```

Reply:
Yes! 30f9e704c